static/js/util/sorting.js

```diff
@@ -1,10 +1,12 @@
 // @flow
```

Comment:
a couple general comment: 

1. i realize this is a somewhat common practice in the codebase right now, but i'm not a fan of us putting specific functionality in a 'util/' directory. if you ask me, these aren't utility functions - they're used by work history and education specifically. any reason that `workEntriesByDate` and `educationEntriesByDate` wouldn't be defined in EmploymentForm.js and EducationDisplay.js respectively? looks like they're the only clients of those functions and i can't see why there would be more
1. can you explain the name `resumeOrder`? it looks like a pretty general sort-by-date function. is it just a legacy name?